cmd/convert.go

```diff
@@ -100,8 +104,12 @@ func init() {
 	// Kubernetes only
 	convertCmd.Flags().BoolVarP(&ConvertChart, "chart", "c", false, "Create a Helm chart for converted objects")
 	convertCmd.Flags().BoolVar(&ConvertDaemonSet, "daemon-set", false, "Generate a Kubernetes daemonset object")
-	convertCmd.Flags().BoolVarP(&ConvertDeployment, "deployment", "d", false, "Generate a Kubernetes deployment object")
+	convertCmd.Flags().BoolVar(&ConvertDeployment, "deployment", false, "Generate a Kubernetes deployment object")
```

Comment:
missed this, but please leave this unmodified, since we are doing backwards compatibility now, leave the `d` in.